internal/core/platform.rs

```diff
@@ -72,6 +72,27 @@ pub trait Platform {
     fn clipboard_text(&self) -> Option<String> {
         None
     }
+
+    /// This function is called when debug() is used in .slint files. The implementation
+    /// should direct the output to some developer visible terminal. The default implementation
+    /// uses stderr if available, or `console.log` when targeting wasm.
+    fn debug_log(&self, _text: &str) {
```

Comment:
I wonder if it would not be nice to take `core::fmt::Arguments` then we might not need to allocate into a string.

Since it is only for debug log, maybe that's not worth it though.

Reply:
Yeah it’s a bit silly how much allocations this produces. I can try tonight maybe, or it could be in a follow-up change.